### DISCUSSION TITLE
How to get the url generated from sulu-link tag

### DISCUSSION THREAD

**Discussion by simontol** [sulu/sulu#7518] [Q&A] [answered]
Hi,
I would like to use the url generated by the sulu-link tag in another section of the template that is not an anchor.
Is it possible to get the same url from a twig function or is there any other way to get a url from a uuid?

**Comment by alexander-schranz** [comment, marked as answer]
@simontol Sulu Links are not resolved inside Twig and so you can not access a resolved URL inside Twig itself. But as always you can create a custom Twig Extension and use the LinkProvider / LinkProviderPool services to resolve a `id/uuid` / `provider` yourself manually.

If you are using the Link Content Type it should already return you a resolved Url.
